src/python/geoclaw/surge/storm.py

```diff
@@ -147,6 +147,10 @@ class Storm(object):
      - *time_offset* (datetime.datetime) A date time that as an offset for the
        simulation time.  This will default to the beginning of the first of the
        year that the first time point is found in.
+     - *wind_speed* (ndarray(:)) Wind speed defined in every record, such as 34kt,
+       50kt, 64kt, etc. Default units are meters/second.
+     - *wind_speed_radius* (ndarray(:)) Radii of every record wind speed. 
+       Default units are meters.
```

Comment:
I would suggest changing this to be a 2D array with one column holding the radii and the other the speeds.  It can then just be called `wind_speeds`.

Reply:
Hi Kyle, I made the requested change.